xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
@faush01 did you find some time to go back and look at "ok so its a DB path resume point issue, I can test this."?
might be the best way to proceed, otherwise we'll have to live with the "IsPlugin-Hack" But it will need to be documented/commented.